zkevm-circuits/src/util/word.rs

```diff
@@ -478,4 +479,16 @@ impl<F: Field> From<WordLegacy<F>> for Word32Cell<F> {
     }
 }
 
+/// Return the hash of the empty code as a Word<Value<F>>
+pub fn empty_code_hash_word_value<F: Field>() -> Word<Value<F>> {
```

Comment:
Here can have simpler and more reusible version

```diff
/// Return the hash of the empty code as a Word<Value<F>>
+ pub fn empty_code_hash_word_value<F: Field>() -> Word<Value<F>> {
+    Word::from(*EMPTY_CODE_HASH).into_value()
+}
```

and have  `From<eth_types::Hash>`

```
+ impl<F: Field> From<eth_types::Hash> for Word<F> {
+    /// Construct the word from u256
+    fn from(value: eth_types::Hash) -> Self {
+        let mut bytes = *value.as_fixed_bytes();
+        bytes.reverse();
+        Word::new([
+            from_bytes::value(&bytes[..N_BYTES_HALF_WORD]),
+            from_bytes::value(&bytes[N_BYTES_HALF_WORD..]),
+        ])
+    }
}
```

Reply:
Done